task_11/css/style.css

```diff
@@ -0,0 +1,719 @@
+body{
+    background: #eceef4;
+    counter-reset: section;
+}
+
+
+/*HEADER*/
+.header{
+    width: 100%;
+    max-width: 1280px;
+    padding: 3em 0;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+}
+.header__logo{
+    width: 100%;
+    max-width: 100px;
+}
+@media screen and (max-width: 1320px),(min-resolution:270dpi){
```

Comment:
If you're using pure css (no preprocessor) consider to use as less @media queries as possible. It will increase rendering performance.
Not for fix, but for future.